### DISCUSSION TITLE
Unicode for Vietnamese cannot be displayed correctly after saved.

### DISCUSSION THREAD

**Discussion by tctruc79** [requarks/wiki#3622] [Error / Bug Report]
### Expected behavior
It must display the Vietnamese unicode correctly.

**Comment by NGPixel** [comment]
Please post your host info as required in the issue template.

**Comment by p1nkv4lkyr13** [comment]
I got the same error with my Wiki app. 

**Host info**
OS: Windows Server 2019 Standard
Node.js: v14.15.1
Database: MSSQL 2012 Express
Wiki.js: v2.5.170

I found out the cause was the data type of column **content** in table **pages**. I changed it from **varchar(MAX)** to **nvarchar(MAX)** to make it able to save Unicode character
